data/random-teams.ts

```diff
@@ -500,8 +500,8 @@ export class RandomTeams {
 		// This space reserved for assorted hardcodes that otherwise make little sense out of context
 		// Landorus
 		this.incompatibleMoves(moves, movePool, 'nastyplot', 'rockslide');
-		// Persian and Seviper
-		this.incompatibleMoves(moves, movePool, 'switcheroo', ['fakeout', 'suckerpunch']);
+		// Persian and Grafaiai
+		this.incompatibleMoves(moves, movePool, 'switcheroo', ['fakeout', 'superfang']);
```

Comment:
I would make this `['trick', 'switcheroo']` to make the rule more general, despite it not actually occurring right now

Reply:
Once we have reason to remove it from the hardcodes section, we'll do that. Thank you, though!